README.md

```diff
@@ -1303,6 +1311,168 @@ task.resume()
 
 **IMPORTANT:** `Graph.QueryError` does not contain user-friendly information. Often, it describes the technical reason for the failure, and shouldn't be shown to the end-user. Handling errors is most useful for debugging.
 
+## Customer Accounts [⤴](#table-of-contents)
+
+Using the Buy SDK, you can build custom storefronts that allow your customers to create accounts, browse orders and manage their information. Since most customer-related actions modify state on the server, they are performed using various `mutation` queries. Let's take a look at a few.
+
+### Creating a customer [⤴](#table-of-contents)
+
+Before a customer can login, they must first create an account. In your application, you can provide a signup form that will run the following `mutation` query. The `input` to the mutation is some basic customer information that will create an account on your shop. Keep in mind that this mutation returns a `Storefront.Customer` object, **not** an access token. After a successful mutation, the customer will still be required to [log in using their credentials](#customer-login-).
+
+```swift
+let input = Storefront.CustomerCreateInput(
+    email:     "john.smith@gmail.com",
+    password:  "123456",
+    firstName: "John",
+    lastName:  "Smith",
+    acceptsMarketing: true
+)
+
+let mutation = Storefront.buildMutation { $0
+    .customerCreate(input: input) { $0
+        .customer { $0
+            .id()
+            .email()
+            .firstName()
+            .lastName()
+        }
+        .userErrors { $0
+            .field()
+            .message()
+        }
+    }
+}
+```
+
+### Customer login [⤴](#table-of-contents)
+
+Any customer with an account can log in to your shop. All login operations are `mutation` queries that exchange customer credentials for an access token. You can log in your customers using the `customerAccessTokenCreate` mutation. Keep in mind that the return access token expires. The expiry `Date` is provided by the `expiresAt` property of the returned payload.
```

Comment:
`mutations` is misleading since `mutation` is a valid keyword. Not `mutations`. I refer to these in the document as `mutation` queries. Although I am open to suggestions.